src/documentation/gtag.ts

```diff
@@ -1,7 +1,7 @@
 'use strict';
 import {GTA3DocumentationProvider, docrequest, CommandDoc, GameDoc, ArgumentDoc} from './interface';
 import {GTA3ScriptController, Command} from '../controller';
-const cheerio = require('cheerio')
+const json = require('jsonify');
```

Comment:
Hmmm, what's the difference between this dep and the runtimes JSON.parse? And it should be added to `rootdir/package.json -> 'dependencies'` as well :)